src/main/java/org/olf/folio/order/OrderImport.java

```diff
@@ -336,12 +336,13 @@ public  JSONArray  upload(String fileName) throws IOException, InterruptedExcept
 				
 				DataField twoFourFive = (DataField) record.getVariableField("245");
 				DataField nineEighty = (DataField) record.getVariableField("980");
+				DataField nineEightyOne = (DataField) record.getVariableField("981");
 			    DataField nineFiveTwo = (DataField) record.getVariableField("952"); 
 			    
 				String title = marcUtils.getTitle(twoFourFive);
 				responseMessage.put("title", title);
 				
-				String notes =  marcUtils.getInternalNotes(nineEighty);
+				String notes =  marcUtils.getInternalNotes(nineEightyOne);
```

Comment:
We'll need to verify with @starsplatter, but

* Internal notes should be mapped to `980$z`
* I don't believe we want to use FOLIO notes for either the internal or receiving notes

> See https://github.com/cul-it/order-import-poc/pull/18#discussion_r606373915